src/tab.js

```diff
@@ -130,7 +130,11 @@ SideTab.prototype = {
     this._setIcon("img/defaultFavicon.svg");
   },
   setSpinner() {
-    this._setIcon("img/loading-spinner.svg");
+    // Workaround until context-stroke is enabled in sidebar by default.
+    const isDarkTheme = document.body.classList.contains("dark-theme");
+    const isShrinked = document.getElementById("tablist").classList.contains("shrinked");
+    const frag = isDarkTheme && isShrinked && "#loading-invert" || "";
```

Comment:
If you switch all the favicons to background images, then it can be handled with CSS alone.
But if you want the favicon as a foreground and put the spinner as a background, both icons will be shown.
http://asamuzak.jp/test/favicon_spinner_test
Note that ::before / ::after pseudo elements won't apply to img element, so you can't overlay the favicon with the spinner.
So you will need to swap favicon to something like dummy transparent image anyway.

Also, even if all the favicons are placed as background images, in that case, it will be necessary to generate CSS dynamically, so it does not seem realistic to me.